### DISCUSSION TITLE
Support for Vlang in competitive programming

### DISCUSSION THREAD

**Discussion by b4apple** [vlang/v#11082] [General]
Rich library, generics, speed (execution and compilation) make V a good candidate for competitive programming. I know language devs might not think of competitive programmers as an ideal target but I seriously enjoyed solving Project Euler problems in V. But no competitive site seems to offer V as a choice. When will we get V support on sites like https://codeforces.com/?

**Comment by medvednikov** [comment]
As someone who did lots of it back in the day, I think it's a perfect target :)

You should contact these sites.

You can also always just submit the C output. Although we need to make -usecache the default first.

**Reply by b4apple** [reply]
I don't enough reputation to convince them ! As a language creator, you'd have much more value to them !
